Easy/ChuckNorris.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import math
+
+# Auto-generated code below aims at helping you parse
```

Comment:
add description,
a lot of magic numbers.
Note that good code should be read like a book.
Try to refactor below (use functions meaningful names etc.)